CandyStorage.cs

```diff
@@ -35,8 +35,15 @@ internal Candy SaveNewCandy(Candy newCandy)
 
             var name = _myCandy[0].Name;
```

Comment:
What are these variables used for?
`name`, `mfc`, `flavor`?

Reply:
I can delete them out and push up again. It's from when I was making sure that the AddCandy() method was working properly. I had these variables writing to the console.